extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -1263,16 +1263,16 @@ static public function getLocalEnvURL( $url, $forcedEnv = null ) {
 		// we do not have valid ssl certificate for these subdomains
 		switch ( $environment ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a "case default" clause to this "switch" statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS131)

[project-id]: wikia:app